java-frontend/src/test/java/org/sonar/java/resolve/SymbolTableTest.java

```diff
@@ -973,7 +973,7 @@ public void Lambdas() throws Exception {
 
 
     JavaSymbol sym = result.symbol("o");
-    assertThat(sym.type.toString()).isEqualTo("!Defered type!"); // unknown while we do the analysis with java 7
+    assertThat(sym.type.toString()).isEqualTo("Object");
```

Comment:
It would be better to use `.is("java.lang.Object")).isTrue();` rather than testing return value of the `toString()` method.